lib/t/matches.ex

```diff
@@ -325,12 +429,41 @@ defmodule T.Matches do
     end)
   end
 
+  defp insert_expired_match(multi) do
+    Multi.run(multi, :insert_expired_match, fn _repo, %{unmatch: unmatch} ->
+      %{id: match_id, users: [user_id_1, user_id_2]} = unmatch
+
+      timestamp = DateTime.utc_now() |> DateTime.truncate(:second) |> DateTime.to_naive()
+
+      {count, _} =
+        Repo.insert_all(
```

Comment:
inserting two ExpiredMatch entries mirroring each other to then delete each of them on "seen" by respective user